setup.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+# flake8: noqa
+
+"""
+Pure Storage Python SDK for FlashArray and VMware Integration
+"""
+
+
+from setuptools import setup, find_packages  # noqa: H301
+
+NAME = 'purepyvmware'
+VERSION = '1.0.0'
+
+REQUIRES = ['purestorage >= 1.18.0', 'pyvmomi >= 6.7.3']
+
+readme = open('README.md', 'r')
+README_TEXT = readme.read()
+readme.close()
+
+setup(
```

Comment:
I notice in here there is no minimum python requirement, but there are f-strings used in the code. These were not introduced until python3.6, so you may want to considering either adding a
```
    python_requires='>=3.6',
```
Or changing the style of the f-strings to previous python3 style
```
string_var.format(variables)
```